overlord/snapshotstate/backend/backend_test.go

```diff
@@ -944,8 +945,10 @@ func (s *snapshotSuite) TestImport(c *check.C) {
 		c.Assert(err, check.IsNil, comm)
 		importingFile := filepath.Join(dirs.SnapshotsDir, fmt.Sprintf("%d_importing", t.setID))
 		if t.inProgress {
-			err = ioutil.WriteFile(importingFile, nil, 0644)
+			flock, err := osutil.NewFileLock(importingFile)
```

Comment:
This should be reverted to `err = ioutil.WriteFile(importingFile, nil, 0644)` now that we don't use locks? (not to confuse anyone who may be looking at these tests later).